libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1638,6 +1654,22 @@ void AP_OSD_Screen::draw_heading(uint8_t x, uint8_t y)
     backend->write(x, y, false, "%3d%c", yaw, SYMBOL(SYM_DEGR));
 }
 
+void AP_OSD_Screen::draw_rrpm(uint8_t x, uint8_t y)
+{
```

Comment:
we need to check for #if AP_RPM_ENABLED and include AP_RPM/AP_RPM_config.h

Reply:
Hi Andrew thank you so much for the recommendations: I have pushed mods in the "fix" commit below, don't know if there's better way to do it though.